pysat/instruments/icon_mighti.py

```diff
@@ -168,132 +213,122 @@ def load(fnames, tag=None, sat_id=None):
     --------
     ::
         inst = pysat.Instrument('icon', 'fuv')
-        inst.load(2019,1)
+        inst.load(2020, 1)
 
     """
 
-    return pysat.utils.load_netcdf4(fnames, epoch_name='EPOCH',
-                                    units_label='Units',
-                                    name_label='Long_Name',
-                                    notes_label='Var_Notes',
-                                    desc_label='CatDesc',
-                                    plot_label='FieldNam',
-                                    axis_label='LablAxis',
-                                    scale_label='ScaleTyp',
-                                    min_label='ValidMin',
-                                    max_label='ValidMax',
-                                    fill_label='FillVal')
-
+    data, mdata = pysat.utils.load_netcdf4(fnames, epoch_name='Epoch',
+                                           units_label='Units',
+                                           name_label='Long_Name',
+                                           notes_label='Var_Notes',
+                                           desc_label='CatDesc',
+                                           plot_label='FieldNam',
+                                           axis_label='LablAxis',
+                                           scale_label='ScaleTyp',
+                                           min_label='ValidMin',
+                                           max_label='ValidMax',
+                                           fill_label='FillVal',
+                                           pandas_format=pandas_format)
+    # xarray can't merge if variable and dim names are the same
+    if 'Altitude' in data.dims:
+        data = data.rename_dims(dims_dict={'Altitude': 'Alt'})
```

Comment:
Alternatively, we can use the more generic form `rename`, which has existed since xarray 0.6.  Since we have not renamed the variables at this point, this should produce the same effect.  I'd still prefer to use the more specific form in 3.0.
```suggestion
        data = data.rename({'Altitude': 'Alt'})
```